psi4/src/psi4/libmints/wavefunction.h

```diff
@@ -574,8 +572,12 @@ class PSI_API Wavefunction : public std::enable_shared_from_this<Wavefunction> {
     SharedMatrix basis_projection(SharedMatrix Cold, Dimension noccpi, std::shared_ptr<BasisSet> old_basis,
                                   std::shared_ptr<BasisSet> new_basis);
 
+    /// Returns the SO basis Lagrangian
```

Comment:
Should be lowercase I believe?

Reply:
I think I agree.  It's [correct to capitalize terms derived from names](http://folk.ntnu.no/haugwarb/Misc/capitalization.pdf) in writing, but I think we want to be as consistent in the API as we can.  An example of the is is the `aotoso` function directly below; here AO and SO should normally be capitalized, but are left lower case.

Reply:
I rename the functions to lowercase, though I did not touch the variable name (still `Lagrangian_`).

Reply:
Thanks @lcyyork - the variable names can be capitalized any way, as long as the API is consistent, which it is now